include/nanvix/config.h

```diff
@@ -28,7 +28,7 @@
 	/**@{*/
 	#define MACHINE_NAME "valhalla" /**< Machine name.              */
 	#define CPU                i386 /**< Cpu model.                 */
-	#define MEMORY_SIZE   0x1000000 /**< Memory size (in bytes).    */
+	#define MEMORY_SIZE   0x4000000 /**< Memory size (in bytes).    */
```

Comment:
@Theldus Should we really change memory size?

Reply:
If I remember correctly, I changed the memory size because the benchmarks I was using used a little over 16MB. You can leave the size as before, there are no problems :)